Cargo.toml

```diff
@@ -157,6 +157,8 @@ default-features = false
 [replace]
 "log:0.3.9" = { git = "https://github.com/busyjay/log", branch = "use-static-module" }
 "log:0.4.6" = { git = "https://github.com/busyjay/log", branch = "revert-to-static" }
+# for test 
```

Comment:
only for test now?

If you just want to do test now, please add WIP or DNM label to your PR

Reply:
Sorry, this comment was used to remind me not to forget to update the Cargo.toml when my PR of "kvproto" is merged, I think this PR can work now.